### PR TITLE
add christoph_hummel.josn

### PR DESCRIPTION
**Pull request by hummelch**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] The JSON file follows the template at [https://github.com/jscraftcamp/website/blob/main/participants/\_template.json](https://jscraftcamp.org/register/) and contains the mandatory fields like `name`, `when`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [ ] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/5/views/1)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

File: participants/christoph_hummel.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	"when": {
+		"friday": true,
+		"saturday": true
+	},
+	"iCanTakeNotesDuringSessions": false,
+	"tags": ["React", "TypeScript", "Storybook", "Next.js"],
+	"vegan": false,
+	"vegetarian": false,
+	"whatIsMyConnectionToJavascript": "In addition to my job as a frontend developer, I use JS for my own projects on a daily basis.",
+	"whatCanIContribute": "Sharing my knowlage about react and next.js",
```
Comment (Narigo) [inline participants/christoph_hummel.json:13]: Typo?

```suggestion
	"whatCanIContribute": "Sharing my knowledge about react and next.js",
```

**Review by Narigo — COMMENTED** [review]
Welcome @hummelch ! Looking forward to seeing you at the JSCC23!
I'm pretty sure there are quite a few people thinking about energy and making IT "green" this year. Sounds like you may be able to share some insights as well when I read about your company?

Let me know if you want to change the registration based on my suggestion or not before we merge it.

Feel free to join [the JSCC Discord](https://discord.gg/3mHhaXVEjP) if you have any questions or want to connect with the broader community 😄

**Comment by hummelch** [comment]
Hi @Narigo,
thanks for fixing my typo. Unfortunately, I can't contribute much on the subject of green IT. As far as I know, this is not really a topic in my department at the moment. But I will ask my colleagues if other teams have something I can share.

**Comment by Narigo** [comment]
No worries, there are lots of things that can be discussed at the JSCC! I did look at website of badenova, so I thought you're all dealing with it the whole time 😅 I didn't realize how big that company is... 😅

I do appreciate if you promote it to your co-workers though because everyone has something to share and learn as we have seen in the past 😅

**Pull request merged**